.changeset/giant-points-build.md

```diff
@@ -0,0 +1,5 @@
+---
+"@accelint/design-toolkit": minor
+---
+
+A refactor introducing breaking changes for the Tooltip component. The structure is noww a light wrapper around the [Tooltip](https://react-spectrum.adobe.com/react-aria/Tooltip.html) implementation. Tooltip has been replaced with Tooltip.Trigger and Tooltip.Body has been replaced with Tooltip. Reference the [docs](https://gohypergiant.github.io/standard-toolkit/?path=/docs/components-tooltip--playground) for implementation instructions.
```

Comment:
Due to this being a breaking change and us having consumers, let's change the version to `major`

NIT: spelling of "noww"